packages/slate-react/src/components/editable.tsx

```diff
@@ -1385,23 +1405,6 @@ const defaultScrollSelectionIntoView = (
   }
 }
 
-/**
```

Comment:
Was this intentionally removed? I’m not seeing the relevance of this change to the rest of the PR.

Reply:
It's dead code, not used anywhere, and not exported. Accidentally committed the removal but I think it's fine to keep it in the pr no?